src/Parser/FileParser.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Spatie\CodeOutline\Parser;
+
+use Spatie\CodeOutline\Elements\EmptyLine;
+use Spatie\CodeOutline\Elements\Line;
+use Spatie\CodeOutline\Elements\Page;
+
+class FileParser implements Parser
+{
+    /** @var string */
+    protected $path;
+
+    public function __construct(string $path)
+    {
+        $this->path = $path;
+    }
+
+    public function getParsed(): Page
+    {
+        $contents = file_get_contents($this->path);
+
+        $lines = explode(PHP_EOL, $contents);
+
+        $page = new Page();
+
+        foreach ($lines as $line) {
+            if (!strlen($line)) {
```

Comment:
I think using the `===0` is the proper way.

Reply:
You're right, thanks!